apps/innbyggerpanelet/src/app/components/misc/propertieValueField/PropertieValueField.tsx

```diff
@@ -0,0 +1,27 @@
+import { Heading } from '@navikt/ds-react'
+import React, { ReactElement, useEffect, useState } from 'react'
+
+import style from './PropertieValueField.module.scss'
+
+export default function PropertieValueField(
```

Comment:
I believe this is a typo, should be Propert**y**ValueField.

Reply:
Oops! I fix